### DISCUSSION TITLE
FR: Show which globals have already been saved to a globals set?

### DISCUSSION THREAD

**Discussion by amphibian** [craftcms/cms#8841] [Ideas]
When managing a site with lots of globals, which need to be organized across multiple global sets (in my example, a multi-lingual site with a zillion strings to translate), it can be difficult to keep track of what has been used, and what goes where.

It would be helpful to know whether a particular field has already been saved into one or more global sets or not. Visually, and maybe even with a tooltip that tells us where it's currently living?

Maybe too niche of a need, but though I'd throw it out there.

<img width="247" alt="Screen Shot 2020-06-12 at 2 09 25 PM" src="https://user-images.githubusercontent.com/53333/84538411-3bc54c80-acb7-11ea-98b6-5a987df70bb9.png">

**Comment by brandonkelly** [comment]
I don’t think we’d want to give them a disabled-looking styling since fields are meant to be reusable, but agree it would be nice to see some additional info about each field, including where else they are being used.
